website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -243,6 +243,23 @@ For more information, see the [`PredictionResult` documentation](https://github.
 For detailed instructions explaining how to build and run a Python pipeline that uses ML models, see the
 [Example RunInference API pipelines](https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/inference) on GitHub.
 
+## Slowly-updating side input pattern to auto update models used in RunInference
+To perform auto updates of the models used in `RunInference` PTransform without stopping the Beam pipeline, pass a [`ModelMetadata`](https://beam.apache.org/releases/pydoc/current/apache_beam.ml.inference.base.html#apache_beam.ml.inference.base.ModelMetadata) side input PCollection to the input parameter `model_metadata_pcoll` of RunInference.
+
+`ModelMetdata` is a `NamedTuple` containing:
+  * `model_id`: Unique identifier for the model. This can be a file path or a URL where the model can be accessed. It is used to load the model for inference. The URL or file path must be in the compatible format so that the respective ModelHandlers can load the models without errors.
+
+    **For example**, `PyTorchModelHandler` loads a model using weights and a model class initially, and during the update, If you pass in weights from a different model class when you update the model via side inputs, it won’t load properly because it’s expecting the weights from the original model class.
```

Comment:
```suggestion
    **For example**, `PyTorchModelHandler` initially loads a model using weights and a model class. If you pass in weights from a different model class when you update the model using side inputs, the model doesn't load properly, because it expects the weights from the original model class.
```